ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -56,6 +56,22 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.peer_groups[peer_group].next_hop_unchanged is arista.avd.defined(true) %}
    neighbor {{ peer_group }} next-hop-unchanged
 {%         endif %}
+{%         if router_bgp.peer_groups[peer_group].remove_private_as.enabled is arista.avd.defined(true) %}
+{%             set remove_private_as_cli = "neighbor " ~ peer_group ~ " remove-private-as" %}
+{%             if router_bgp.peer_groups[peer_group].remove_private_as.all is arista.avd.defined(true) %}
+{%                 set remove_private_as_cli = remove_private_as_cli ~ " all" %}
+{%             endif %}
+   {{ remove_private_as_cli }}
+{%             if router_bgp.peer_groups[peer_group].remove_private_as.ingress is arista.avd.defined(true) %}
```

Comment:
Same as above. If only `replace_as` is true, we can still build the Cli. Same for docs. And need some clarification on what will have priority if `enabled: false` and `replace_as: true`. Otherwise just consider an alternative data model

Reply:
This would not match other areas in AVD. it is fine to require enabled: true and just ignore the rest if not set correctly.

Reply:
ack. Let's focus on the data model anyway, as if we change it the problem may be fixed automatically